receiver/aerospikereceiver/metadata.yaml

```diff
@@ -0,0 +1,136 @@
+    enum:
+      - data
+      - index
+      - set_index
+      - sindex
+  scan_type:
+    value: type
+    description: Type of scan operation performed on a namespace
+    enum:
+      - aggr
+      - basic
+      - ops_bg
+      - udf_bg
+  scan_result:
+    value: result
+    description: Result of a scan operation performed on a namespace
+    enum:
+      - abort
+      - complete
+      - error
+  transaction_type:
+    value: type
+    description: Type of transaction performed on a namespace
+    enum:
+      - delete
+      - read
+      - udf
+      - write
+  transaction_result:
+    value: result
+    description: Result of a transaction performed on a namespace
+    enum:
+      - error
+      - filtered_out
+      - not_found
+      - success
+      - timeout
+  connection_type:
+    value: type
+    description: Type of connection to an Aerospike node
+    enum:
+      - client
+      - fabric
+      - heartbeat
+  connection_op:
+    value: op
+    description: Operation performed with a connection (open or close)
+    enum:
+      - close
+      - open
+
+metrics:
+  aerospike.node.memory.free:
+    enabled: true
+    description: Percentage of the node's memory which is still free
+    extended_documentation: Aerospike Metric system_free_mem_pct
+    unit: '%'
+    gauge:
+      value_type: int
+  aerospike.node.connection.count:
+    enabled: true
+    description: Number of connections opened and closed to the node
+    extended_documentation: Aggregate of Aerospike Metrics client_connections_closed, client_connections_opened, fabric_connections_closed, fabric_connections_opened, heartbeat_connections_closed, heartbeat_connections_opened
+    unit: '{connections}'
+    attributes: [connection_type, connection_op]
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+  aerospike.node.connection.open:
+    enabled: true
+    description: Number of open connections to the node
+    extended_documentation: Aggregate of Aerospike Metrics client_connections, fabric_connections, heartbeat_connections
+    unit: '{connections}'
+    attributes: [connection_type]
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+  aerospike.namespace.memory.usage:
+    enabled: true
+    description: Memory currently used by each component of the namespace
+    extended_documentation: Aggregate of Aerospike Metrics memory_used_data_bytes, memory_used_index_bytes, memory_used_set_index_bytes, memory_used_sindex_bytes
+    unit: By
+    attributes: [namespace_component]
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+  aerospike.namespace.memory.free:
+    enabled: true
+    description: Percentage of the namespace's memory which is still free
+    extended_documentation: Aerospike metric memory_free_pct
+    unit: '%'
+    gauge:
+      value_type: int
+  aerospike.namespace.disk.available:
```

Comment:
Yeah I used `available` here because another Aerospike metric is `disk_free` which doesn't account for fragmentation. I can change it for consistency, though, and clarify that in the `extended_documentation` https://docs.aerospike.com/reference/metrics#device_available_pct

Reply:
Since there's prior art, I think it's fine to leave it. Just didn't want entirely arbitrary variation.